docker/conf/log.config

```diff
@@ -16,14 +16,11 @@ handlers:
     filters: [context]
 
 loggers:
-    synapse:
-        level: {{ SYNAPSE_LOG_LEVEL or "WARNING" }}
-
     synapse.storage.SQL:
         # beware: increasing this to DEBUG will make synapse log sensitive
         # information such as access tokens.
-        level: {{ SYNAPSE_LOG_LEVEL or "WARNING" }}
+        level: INFO
```

Comment:
Why isn't this `{{ SYNAPSE_LOG_LEVEL or "INFO" }}`?

Reply:
We don't want it to go to DEBUG if we set Synapse to DEBUG. If they want to increase verboseness here, they'll have to use a proper config.